diff -p -up dotconf-1.0.13/dotconf.m4.automake dotconf-1.0.13/dotconf.m4
--- dotconf-1.0.13/dotconf.m4.automake	2008-03-13 17:07:48.000000000 +0100
+++ dotconf-1.0.13/dotconf.m4	2008-03-13 17:08:04.000000000 +0100
@@ -2,7 +2,7 @@
 dnl ## AM_PATH_DOTCONF([MINIMUM_VERSION, [ACTION-IF-FOUND [, ACTION-IF-NOT_FOUND]]])
 dnl ##
 dnl ##
-AC_DEFUN(AM_PATH_DOTCONF,
+AC_DEFUN([AM_PATH_DOTCONF],
 [dnl
 dnl Get the cflags and libs from the dotconf-config script
 dnl
